content/sensu-go/5.18/guides/securing-sensu.md

```diff
@@ -15,13 +15,14 @@ menu:
 - [Secure the API and dashboard](#secure-the-api-and-dashboard)
 - [Secure Sensu agent-to-server communication](#secure-sensu-agent-to-server-communication)
 - [Sensu agent TLS authentication](#sensu-agent-tls-authentication)
-- [Create self-signed certificates](#create-self-signed-certificates)
-- [Next steps](#next-steps)
+- [Next step: Run a Sensu cluster](#next-step-run-a-sensu-cluster)
 
 As with any piece of software, it is critical to minimize any attack surface the software exposes.
 Sensu is no different.
 This guide describes the components you need to secure to make Sensu production-ready.
 
+To use this guide, you must have previously [generated the certificates][1] you will need to secure Sensu in production.
```

Comment:
`that you will need for securing Sensu in production` ?

The sentence itself sounds off to me.

Reply:
```suggestion
Before you can use this guide, you must have [generated the certificates][1] you will need to secure Sensu.
```

Reply:
@Blue0ctober WDYT? ☝️

Reply:
LGTM